extensions/WEBGL_color_buffer_float/extension.xml

```diff
@@ -38,10 +38,21 @@
       </feature>
 
       <feature>
+        <p>The 32-bit floating-point type <code>RGB32F</code> may become available
+        as a color-renderable format. Renderbuffers can be created in this
+        format. These and textures created with <code>format = RGB</code> and
+        <code>type = FLOAT</code> as specified in <a
+        href="http://www.khronos.org/registry/webgl/extensions/OES_texture_float/">OES_texture_float</a>,
+        can be attached to framebuffer object color attachments for rendering
+	if <code>checkFrameBufferStatus()</code> returns <code>FRAMEBUFFER_COMPLETE</code>.
```

Comment:
Capitalization should be:
checkFramebufferStatus()

Reply:
Thanks for catching it. Updated.